locale/de/locale.cfg

```diff
@@ -23,13 +24,13 @@ hide_done=Verberge abgeschlossene Aufgaben
 todolist-toggle-ui=Todo Liste minimieren/maximieren
 
 [mod-setting-name]
-todolist-show-minimized=Zeige "Todo List" Schalter wenn minimiert.
+todolist-show-button=Zeige "Todo List" Schalter ins der mod ui.
```

Comment:
I suggest
`Zeige "Todo List" Schalter an/verstecke ihn`

Reply:
If I'm reading that correctly (with my rather poor german augmented by google translate), it seems to translate to 'Show/Hide "Todo List" button'.  Maybe we just make them all say their equivalent of 'Show "Todo List" button', without mentioning hiding or the location.

I'm thinking that, because it's a checkbox/boolean value, the label should be descriptive of the 'on' state, with the state of the checkbox providing the polarity(?) of the setting (as in checked = 'this statement is true', unchecked = 'negate this statement')